contracts/universalSchemes/UpgradeScheme.sol

```diff
@@ -34,7 +34,7 @@ contract UpgradeScheme is UniversalScheme, ExecutableInterface {
     }
 
     // A mapping from the organization's (Avatar) address to the saved data of the organization:
-    mapping(address=>mapping(bytes32=>UpgradeProposal)) public organizationsProposals;
+    mapping(address=>mapping(bytes32=>UpgradeProposal)) public organizationProposals;
```

Comment:
please revert it back to organizationsProposals

Reply:
I'm not sure where this diff is coming from, it is named `organizationProposals` in the code.

Reply:
It should be name organizationsProposals